test/spinach/cli_test.rb

```diff
@@ -257,6 +257,15 @@
       end
     end
 
+    describe 'when a particular feature list is passed with multiple lines' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.